code/mathematical-algorithms/reverse_number/reverse.c

```diff
@@ -0,0 +1,14 @@
+#include<stdio.h>
+int main()
+{
+	int num, i = 0;
+	printf("Enter number to be reversed: \n");
+	scanf("%d", &num);
+	
+	printf("Reverse of number %d is: ", num);
+	while(num > 0) {
```

Comment:
Could you also handle negative case?
Thanks.

Reply:
Yes sure, so should I convert -1234 to -4321?